README.md

```diff
@@ -6,11 +6,61 @@ This project provides an API for storing and retrieving learning resources that
+If this is your first project using GitHub, Python, or pip follow these steps first:
+1. Install [Git](https://git-scm.com/downloads).
+- Choose your OS from the website and follow the prompts.  This installs Git and the Bash Terminal on your machine.
+- Extra: [Git Documentation](https://git-scm.com/doc) for more information on Git.
+2. Install and Setup Python.
+- [Windows Users](https://docs.python.org/3/using/windows.html)
+- [Mac Users](https://docs.python.org/3/using/mac.html)
+3. Install [pip](https://pip.pypa.io/en/stable/installing/).
+4. Sometimes these installs can be tricky.  If you get stuck ask for help in the Slack [#oc-python-projects](https://operation-code.slack.com/messages/C7NJLCCMB) channel! 
+5. [Fork](https://help.github.com/articles/fork-a-repo/) this repository.
+6. [Clone](https://help.github.com/articles/cloning-a-repository/) this repository in the Git Bash terminal.
+- Navigate to where you would like to clone a fork of your repository. `cd \path\to\directory`
+- Clone your fork of the project `git clone {project_url} resources`
 
-### Windows setup
 
-Coming Soon!
+Next, you'll need to configure a database. Once this project is deployed, we'll be using [PostgreSQL](https://www.postgresql.org/docs/). To set up a psql instance locally, follow the instructions for your OS:
+
+### Windows Setup
+
+1. Setup up paths and system variables:   
+- Search --> Edit the [system environment variables](https://docs.microsoft.com/en-us/windows/desktop/shell/user-environment-variables) --> Environment Variables... --> Path --> Edit... --> New --> Enter `C:\ProgramData\chocolatey\bin` --> OK
+- New --> `C:\Program Files\PostgreSQL\10\bin` --> OK
+2. Set the System Variables.
+- Click the bottom New... -->
+- Connection Variable:
+    - Variable Name: `SQLALCHEMY_DATABASE_URI`
+    - Variable Value: `postgresql://aaron:password@127.0.0.1:5432/resources` (instead of aaron and password put what user name and password you want to use for postgresql) 
+- FLASK_APP Variable:
+    - Variable name: `FLASK_APP`
+    - Variable Value: `run.py`
+    - Optionally, enable debugging by setting the environment to development with the Variable Value: `development`
+3. Close out of Environment Variables and System Properites.
+4. Start your administrative shell. `ctrl + x` --> Windows Powershell (Admin).
+- **Make sure you use the _admin powershell_ throughout all the steps.**
+5. [Install Chocolatey](https://chocolatey.org/docs/installation#installing-chocolatey) if you do not already have it installed.
+6. In your powershell (admin), run `choco install postgresql10`. Follow the prompts.
+7. Upgrade postgresql: `choco upgrade postgresql`.
+8. Start postgres: `psql -U postgres` you will see postgres-# in the terminal.
+9. Create your user with `CREATE USER name;` The terminal will print CREATE ROLE.
```

Comment:
Need to add a password here. The command should be `CREATE USER name WITH PASSWORD 'password';`